src/main/resources/maven_templates/maven-assembly-plugin.xml

```diff
@@ -0,0 +1,21 @@
+<plugin>
+    <artifactId>maven-assembly-plugin</artifactId>
+    <groupId>org.apache.maven.plugins</groupId>
+    <version>3.3.0</version>
+    <configuration>
+        <descriptors>
+            <descriptor>assembly/all-dependencies.xml</descriptor>
+        </descriptors>
+        <finalName>NAME_OF_YOUR_JAR</finalName>
+        <appendAssemblyId>false</appendAssemblyId>
+    </configuration>
+    <executions>
```

Comment:
I think we have different configurations and executions depending on the project for this plugin. I don't want to have them changed after I run the project-keeper, so please, check the different projects and test it

Reply:
Release Robot is different for example